### PR TITLE
Better readme

### PR DESCRIPTION
**Pull request by Ybalrid**
Hi again, 

I added badges that show the latest build status on Travis and AppVeyor, the overview of the project from the website, and a link to it. 

The name of the repository is now the top level title on the file. This makes  a bit more sense to do it this way IMO. 

This is, again, just to make the presentation of the project looking a bit nicer. :smile:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
+`master` branch CI status :  [![Build Status](https://travis-ci.org/kcat/openal-soft.svg?branch=master)](https://travis-ci.org/kcat/openal-soft) [![Windows Build Status](https://ci.appveyor.com/api/projects/status/github/kcat/openal-soft?branch=master&svg=true)](https://ci.appveyor.com/api/projects/status/github/kcat/openal-soft?branch=master&svg=true)
 
 OpenAL Soft is an LGPL-licensed, cross-platform, software implementation of the OpenAL 3D audio API. It's forked from the open-sourced Windows version available originally from openal.org's SVN repository (now defunct).
 OpenAL provides capabilities for playing audio in a virtual 3D environment. Distance attenuation, doppler shift, and directional sound emitters are among the features handled by the API. More advanced effects, including air absorption, occlusion, and environmental reverb, are available through the EFX extension. It also facilitates streaming audio, multi-channel buffers, and audio capture.
 
-More information available on the [official website](http://openal-soft.org/)
+More information are available on the [official website](http://openal-soft.org/)
```
Comment (kcat) [inline README.md]: I think "is" would be a better word choice there than "are". Also, a better header title would be OpenAL Soft, rather than OpenAL-soft.

Comment (Ybalrid) [reply README.md]: Oh, that's my stupid brain thinking in French while writing English again. I'm going to fix that, and remove the '-'

**Comment by Ybalrid** [comment]
I just pushed the changes

**Comment by kcat** [comment]
Thanks for this, by the way. :)

**Comment by Ybalrid** [comment]
My pleasure :wink:
That was nothing. I'm thanking you for actively developing and maintaining this library.

**Pull request merged**